index.html

```diff
@@ -16,11 +16,12 @@
         <header class="header">
           <p class="logo">Welcome</p>
           <ul class="nav-list">
-            <li class="nav-item"><a href="#Welcome">Portofolio</a></li>
-            <li class="nav-item"><a href="#About">About</a></li>
-            <li class="nav-item"><a href="#Contact">Contact</a></li>
+            <li class="nav-item close-menu"><a href="#Welcome">Portofolio</a></li>
```

Comment:
When I click on the `portfolio` link in the navbar, it doesn't exactly go the the `portfolio` section. Kindly fix this issue and then submit it for a re-review.